packages/test/test-end-to-end-tests/src/test/gc/gcDatastoreAliased.spec.ts

```diff
@@ -43,6 +43,8 @@ describeFullCompat("GC Data Store Aliased", (getTestObjectProvider) => {
         const ds1 = await requestFluidObject<ITestDataObject>(aliasableDataStore1, "");
 
         (aliasableDataStore1 as any).fluidDataStoreChannel.bindToContext();
+        // yield a turn to ensure we get to flush the attach op
+        await new Promise<void>((resolve) => setTimeout(resolve));
```

Comment:
Curious, why is this required? I.e. how change in behavior results in this change? Ops are flushed async either way, so I'd expect not requiring this change?
This suggest that change is breaking, i.e. it cases change in behavior that clients might need to react to.

Reply:
This is not required. It's actually overkill. Will remove it.

Reply:
Leftover from testing, sorry..